docs/articles/Products/ProductDefinition.md

```diff
@@ -12,15 +12,15 @@ The level 1 product group definition is done by creating a class used to represe
 
```

Comment:
Which classes should be linked here best instead of `Samples.WatchProduct` and `Samples.NeedleProduct`?